modules/nf-core/purecn/run/meta.yml

```diff
@@ -0,0 +1,105 @@
+name: "purecn_run"
+description: Run PureCN workflow to normalize, segment and determine purity and ploidy
+keywords:
+  - copy number alteration calling
+  - hybrid capture sequencing
+  - targeted sequencing
+  - DNA sequencing
+tools:
+  - "purecn":
+      description: "Copy number calling and SNV classification using targeted short read sequencing"
+      homepage: "https://bioconductor.org/packages/release/bioc/html/PureCN.html"
+      documentation: "https://bioconductor.org/packages/release/bioc/html/PureCN.html"
+      tool_dev_url: "https://github.com/lima1/PureCN"
+      doi: "10.1186/s13029-016-0060-z"
+      licence: "Artistic-2.0"
+      args_id: "$args"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - intervals:
+      type: file
+      description: |
+        BED file of target intervals, generated from IntervalFile.R
+      pattern: "{*.bed,*.txt}"
+  - coverage:
+      type: file
+      description: Coverage file generated from Coverage.R
+      pattern: "*.txt"
+  - normaldb:
+      type: file
+      description: |
+        Normal panel in RDS format, generated from NormalDB.R
+      pattern: "*.rds"
+  - genome:
+      type: string
+      description: Genome build
+
+output:
+  - pdf:
+      type: file
+      description: |
+        PDF file containing copy number plots
+      pattern: "*.pdf"
+  - local_optima_pdf:
+      type: file
+      description: |
+        PDF file containing local optima plots
+      pattern: "*_local_optima.pdf"
+  - seg:
+      type: file
+      description: |
+        Segmentation file generated from DNAcopy.R
+      pattern: "*_dnacopy.seg"
+  - genes_csv:
+      type: file
+      description: |
+        CSV file containing gene copy number calls
+      pattern: "*_genes.csv"
+  - amplification_pvalues_csv:
+      type: file
+      description: |
+        CSV file containing amplification p-values
+      pattern: "*_amplification_pvalues.csv"
+  - vcf_gz:
+      type: file
+      description: |
+        GZipped VCF file containing SNV calls
+      pattern: "*.vcf.gz"
+  - variants_csv:
+      type: file
+      description: |
+        CSV file containing SNV calls
+      pattern: "*_variants.csv"
+  - loh_csv:
+      type: file
+      description: |
+        CSV file containing LOH calls
+      pattern: "*_loh.csv"
+  - chr_pdf:
+      type: file
+      description: |
+        PDF file containing chromosome plots
+      pattern: "*_chromosomes.pdf"
+  - segmentation_pdf:
+      type: file
+      description: |
+        PDF file containing segmentation plots
+      pattern: "*_segmentation.pdf"
+  - multisample.seg:
+      type: file
+      description: |
+        Segmentation file generated from DNAcopy.R
```

Comment:
Change to "Tab-delimited file containing segmentation data from multiple samples". DNAcopy.R is misleading given that multiple segmentation algorithms can be used here.